docs/advanced-config/README.md

```diff
@@ -99,7 +99,7 @@ Style strings are a list of words, separated by whitespace. The words are not ca
 
 where `<color>` is a color specifier (discussed below). `fg:<color>` and `<color>` currently do the same thing , though this may change in the future. The order of words in the string does not matter.
 
-The `none` token overrides all other tokens in a string, so that e.g. `fg:red none fg:blue` will still create a string with no styling. It may become an error to use `none` in conjunction with other tokens in the future.
+The `none` token overrides all other tokens in a string if it is not part of a `bg:` specifier, so that e.g. `fg:red none fg:blue` will still create a string with no styling. `bg:none` does nothing so `fg:red bg:none` is equivalent to `red` or `fg:red`. It may become an error to use `none` in conjunction with other tokens in the future.
```

Comment:
Is the second change here correct? My impression was that `bg:none` explicitly sets a "transparent" background (or rather, a background that matches the one in the rest of the terminal).

Reply:
This is indeed incorrect, I forgot to update it in my second fix I did. I have now updated it to match the current behavior.